scripts/build-tools.sh

```diff
@@ -146,12 +146,20 @@ main()
                 exit 0
         fi
 
-        # Keep 'topologies' first because it's the noisiest.
-        for util in topologies tests; do
-                if eval '$DO_BUILD_'$util; then
-                        make_tool $util
-                fi
-        done
+        if eval $DO_BUILD_topologies; then
```

Comment:
This is suspicious, does this really work? I would expect this new code to perform like this:

```
eval $DO_BUILD_topologies;
eval _topologies;

=> `_topologies` command not found, return `false` always.
```

I would have expected the original quotes to be preserved: 
```suggestion
        if eval '$DO_BUILD_'topologies; then
```

While `eval` may stretch `shellcheck` a bit, it's always, always worth asking `shellcheck` what it thinks.

Reply:
Sorry, my bad. I shouldn't review after a certain hour.

The `eval` is not required anymore, just drop it. It's confusing and confused me.

```suggestion
        if ${DO_BUILD_topologies}; then
```
Also run `shellcheck -x` - always.